content/100.figure-table-legends.md

```diff
@@ -210,3 +227,19 @@ B. This example heatmap from the cell type summary report compares submitter-pro
 This heatmap is only shown in the cell type summary report if submitters provided cell type annotations.
 Heatmap cells are colored by the Jaccard similarity index.
 A value of 1 means that there is complete overlap between which cells are annotated with the two labels being compared, and a value of 0 means that there is no overlap between which cells are annotated with the two labels being compared.
+
+<!-- Figure S6 -->
+![**Consensus cell type annotation gene expression in other diagnosis groups.**](https://raw.githubusercontent.com/AlexsLemonade/scpca-paper-figures/main/figures/compiled_figures/pngs/figure_s6.png?sanitize=true){#fig:figS6 tag="S6" width="9in"}
+
+Panels A-C show dot plots of marker gene expression across broad consensus cell type annotation groups, specifically considering cells across ScPCA diagnosis groups Leukemia (A), Sarcoma (B) and Other solid tumors (C).
+In each panel, the total number of cells in each cell type annotation group across samples is shown in parentheses along the y-axis.
+The x-axis displays marker genes for each cell type annotation group.
+Dots are colored by mean gene expression across all ScPCA libraries and sized proportionally to the percent of libraries they are observed in.
```

Comment:
Just noting whatever change we make for 5A in describing marker genes should be repeated here.